force_bdss/bdss_application.py

```diff
@@ -65,7 +67,8 @@ def _load_failure_callback(plugins, manager, entry_point, exception):
     Reports failure to load a module through stevedore, using the
     on_load_failure_callback option.
     """
-    logging.error(
+    log.error(
```

Comment:
if you want you can combine the `log.error` and `log.exception` by adding a kwarg `exc_info=True` to `log.error`.